hat/assets/js/apps/Iaso/domains/orgUnits/reviewChanges/Components/ReviewOrgUnitChangesButtons.tsx

```diff
@@ -37,6 +37,7 @@ export const ApproveOrgUnitChangesButtons: FunctionComponent<Props> = ({
     }, [isNewOrgUnit, changeRequest, selectedFields]);
 
     const [isApproved, setIsApproved] = useState<boolean>(false);
+    const [isRejected, setIsRejected] = useState<boolean>(false);
```

Comment:
You can either remove `isApproved` or use it instead of `isRejected`, since it's not used anywehere anymore, or the checks using it where actually testing for rejection